cli/utils/utils.go

```diff
@@ -4,8 +4,19 @@ import (
 	"errors"
 	"os"
 	"path/filepath"
+        "bufio"
```

Comment:
Weird indent :D But might be just github's display

Reply:
vim 🤦